### PR TITLE
feat(audit): add ability to ignore Github Security Advisory

### PR DESCRIPTION
**Pull request by IanKrieger**
Resolves https://github.com/pnpm/pnpm/issues/6838

Adds `ignoreGhsas` config option so that you can also target a Github Security Advisory to be ignored
```
  auditConfig: {
    ignoreGhsas: [
      'GHSA-42xw-2xvc-qx8m',
      'GHSA-4w2v-q235-vp99',
      'GHSA-cph5-m8f7-6c5x',
      'GHSA-vh95-rmgr-6w4m',
    ],
  },
```

### CODE REVIEW

File: pnpm/src/main.ts [hunk 1]
```diff
@@ -74,8 +74,10 @@ export async function main (inputArgv: string[]): Promise<void> {
     if (unknownOptionsArray.every((option) => DEPRECATED_OPTIONS.has(option))) {
       let deprecationMsg = `${chalk.bgYellow.black('\u2009WARN\u2009')}`
       if (unknownOptionsArray.length === 1) {
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
         deprecationMsg += ` ${chalk.yellow(`Deprecated option: '${unknownOptionsArray[0]}'`)}`
       } else {
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```
Comment (zkochan) [inline pnpm/src/main.ts]: why were this comments added? This file wasn't modified.

**Review by zkochan — COMMENTED** [review]
add changesets

**Comment by IanKrieger** [comment]
The pre-commit hooks kept failing on these lines specifically, even though
I did not touch them. I added the ignore so I could push the PR.

File: pnpm/src/main.ts (hunk 1 above)
Comment (IanKrieger) [reply pnpm/src/main.ts]: The pre-commit hooks kept failing on these lines specifically, even though I did not touch them. I added the ignore so I could push the PR.

Comment (zkochan) [reply pnpm/src/main.ts]: ok, well you should fix the warning then. Not ignore it. Maybe something like 

```
deprecationMsg += ` ${chalk.yellow(`Deprecated option: '${unknownOptionsArray[0] as string}'`)}`
```

Comment (IanKrieger) [reply pnpm/src/main.ts]: Sure, done

**Comment by zkochan** [comment]
please, add changesets as described in contributing.md

**Review by zkochan — APPROVED** [review]

**Pull request merged**